lang/cs/Org.Apache.REEF.IMRU.Examples/PipelinedBroadcastReduce/BroadcastReceiverReduceSenderMapFunction.cs

```diff
@@ -32,9 +34,16 @@ internal sealed class BroadcastReceiverReduceSenderMapFunction : IMapFunction<in
 
         private int _iterations;
 
+        /// <summary>
+        /// The reference of the ITaskState. 
+        /// Map function can use it to save/restore task state at context level if it needs. 
+        /// </summary>
+        private readonly ITaskState _taskState;
+
         [Inject]
-        private BroadcastReceiverReduceSenderMapFunction()
+        private BroadcastReceiverReduceSenderMapFunction(ITaskState taskState)
```

Comment:
I have removed it from the places we don't need to use and from original IMRU examples and only apply it in FT examples.